dts/bindings/ethernet/snps,dwcxgmac.yaml

```diff
@@ -0,0 +1,467 @@
+# Copyright(c) 2023 Intel Corporation.
+# SPDX - License - Identifier : Apache - 2.0
+
+compatible: "snps,dwcxgmac"
+
+description: Synopsys DesignWareCore XGMAC
+
+include:
+  - name: reset-device.yaml
+  - name: ethernet-controller.yaml
+
+properties:
+  reg:
+    required: true
+  interrupts:
+    required: true
+  max-frame-size:
+    type: int
+    default: 1500
+    description: |
+      Maximum ethernet frame size.  The current ethernet frame sizes
+      supported by hardware are standard and jumbo (up to 16KB) frames. This
+      means that normally xgmac will reject any frame above max-frame-size
+      value.  The default value is 1518, which represents an usual
```

Comment:
the default value is 1500 not 1518? and also would this be better as an enum ("standard", "jumbo") and should it be in the generic yaml?

Reply:
Jumbo packet length can be any value more than the standard frame size of 1518 and can be up to 16KB for dwcXGMAC.  Because of this reason, I did define it as an integer.